src/app/modules/main/chat_section/controller.nim

```diff
@@ -355,6 +355,17 @@ proc init*(self: Controller) =
   self.events.on(SIGNAL_MAILSERVER_HISTORY_REQUEST_COMPLETED) do(e:Args):
     self.delegate.setLoadingHistoryMessagesInProgress(false)
 
+  if self.isCommunitySection:
```

Comment:
We already have this condition at line 164. You can just put the event in the same `if`